File: app/src/main/java/tech/leonam/resultadodocertame/service/ServiceSalvaProvas.java
package tech.leonam.resultadodocertame.service;

import android.content.Context;

import tech.leonam.resultadodocertame.model.entidade.ConfigProva;
import tech.leonam.resultadodocertame.model.interfaces.InterfaceSalvaProvas;
import tech.leonam.resultadodocertame.model.repository.SalvaProva;

public class ServiceSalvaProvas implements InterfaceSalvaProvas {
    @Override
    public boolean salvaProvas(Context context, ConfigProva configProva) {
        return new SalvaProva().salvaProvas(context,configProva);
    }
}

File: app/src/main/java/tech/leonam/resultadodocertame/modelView/ServiceSalvaProvas.kt
package tech.leonam.resultadodocertame.modelView

import android.content.Context
import tech.leonam.resultadodocertame.model.entidade.ConfigProva
import tech.leonam.resultadodocertame.model.interfaces.InterfaceSalvaProvas
import tech.leonam.resultadodocertame.model.repository.SalvaProvaDao

class ServiceSalvaProvas : InterfaceSalvaProvas {
    @Throws(Exception::class)
    override fun salvaProvas(context: Context?, configProva: ConfigProva?): Boolean {
        return SalvaProvaDao().salvaProvas(context, configProva)
    }
}